pyvrp/read.py

```diff
@@ -83,41 +84,43 @@ def read(
 
     # A priori checks
     if "dimension" in instance:
-        num_clients = instance["dimension"]
+        dimension: int = instance["dimension"]
     else:
         if "demand" not in instance:
             raise ValueError("File should either contain dimension or demands")
-        num_clients = len(instance["demand"])
+        dimension = len(instance["demand"])
+
+    depots: np.ndarray = instance.get("depot", np.array([0]))
+    num_vehicles: int = instance.get("vehicles", dimension - 1)
+    capacity: int = instance.get("capacity", _INT_MAX)
 
-    depots = instance.get("depot", np.array([0]))
-    num_vehicles = instance.get("vehicles", num_clients - 1)
-    capacity = instance.get("capacity", _INT_MAX)
-    edge_weight = round_func(instance["edge_weight"])
+    edge_weight: np.ndarray = round_func(instance["edge_weight"])
+    distances = edge_weight  # distance and duration are assumed to be the same
```

Comment:
Technically this comment does not hold for CVRP (I don't mind)

Reply:
Good catch, I've updated the code a bit to make this more obvious, by moving the durations assignment down to where we check time windows.